rx-groovy/src/main/java/io/vertx/rx/groovy/RxHelper.java

```diff
@@ -76,4 +77,50 @@ public static RxJavaSchedulersHook schedulerHook(Context context) {
       }
     };
   }
+
+  /**
+   * Returns a json unmarshaller for the specified java type as a {@link rx.Observable.Operator} instance.<p/>
```

Comment:
IIUC it's not just json but any format the provided mapper is configured to work with

Reply:
Good point